packages/internals/src/engine-commands/getEngineVersion.ts

```diff
@@ -8,6 +7,10 @@ import * as TE from 'fp-ts/TaskEither'
 import { resolveBinary } from '../resolveBinary'
 import { loadLibrary } from '../utils/load'
 
+type NodeAPILibrary = {
```

Comment:
We could've moved whole library type definition to internals, but since it is just one function, I've decided it is not worth it.